components/multiselect/custom-value.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Custom Value
```

Comment:
```suggestion
title: Custom Values
```
I suggest using plural here, too. The MultiSelect generally supports multiple selection and can have multiple custom values.